website/server/libs/amazonPayments.js

```diff
@@ -252,6 +252,13 @@ api.subscribe = async function subscribe (options) {
     if (!result) throw new NotAuthorized(i18n.t('invalidCoupon'));
   }
 
+  let amount = sub.price;
+
+  if (groupId) {
+    let group = await Group.findById(groupId).exec();
+    amount = sub.price + (group.memberCount - 1) * 3;
```

Comment:
These look like magic numbers to me. Could we save these as variables so it's easier to reason what they mean?